projects/plugins/jetpack/_inc/lib/class.jetpack-keyring-service-helper.php

```diff
@@ -98,13 +117,14 @@ function get_services( $filter = 'all' ) {
 
 	/**
 	 * Gets a URL to the public-api actions. Works like WP's admin_url
+	 * on WordPress.com this is/calls Keyring::admin_url
```

Comment:
This reads like one sentence, but I think you meant two separate sentences.
```suggestion
	 * Gets a URL to the public-api actions. Works like WP's admin_url.
	 * On WordPress.com this is/calls Keyring::admin_url.
```